Automattic/wp-calypso
# Pricing page: add product category filter

### Changes proposed in this Pull Request

This PR adds a category filter to the _More Products_ section of the pricing page on mobile and tablet.

### Testing instructions

- Download this PR and run cloud or Calypso
- Visit the pricing/plans page
- Check that the page looks and behaves as in production on desktop (viewport width > 960px)
- Check that you can see the category filter on mobile and tablet, and that it behaves as expected

### Screenshots
<img width="475" alt="Screen Shot 2021-12-13 at 4 01 34 PM" src="https://user-images.githubusercontent.com/1620183/145888127-688c480c-d30e-4b33-a0b6-9215efef2361.png">
